MembraneRTC/Sources/MembraneRTC/Types/TrackContext.swift

```diff
@@ -50,12 +50,12 @@ public class TrackContext {
     /// - when sender stopped sending some encoding (because of bandwidth change)
     /// - when receiver doesn't have enough bandwidth
     /// Some of those reasons are indicated in TrackContext.encodingReason
-    public func setOnTrackEncodingChangeListener(listener: ((_ trackContext: TrackContext) -> Void)?) {
+    public func setOnEncodingChangeListener(listener: ((_ trackContext: TrackContext) -> Void)?) {
```

Comment:
```suggestion
    public func setOnEncodingChangedListener(listener: ((_ trackContext: TrackContext) -> Void)?) {
```

Could we also adjust our internal/private variable names?